hubmap_commons/hm_auth.py

```diff
@@ -465,6 +465,8 @@ def getUserTokenFromRequest(self, httpReq, getGroups):
             return tokenResp        
     
     def getUserInfo(self, token, getGroups = False):
+        if token is None or token.strip() == '':
+            return Response("Auth token required", 400)
```

Comment:
Should return the [auth error messages from `getAuthorizationTokens()`](https://github.com/hubmapconsortium/commons/blob/1ca9777c70d88c02904b2a5dd185a9ce52f84cfb/hubmap_commons/hm_auth.py#L386-L400) when `instance(token, Response)`.

Also it's better to use status code 401 to be precise since this is more about authentication.